### DISCUSSION TITLE
Problem with repo and google cloud build

### DISCUSSION THREAD

**Discussion by inoovum** [community/community#40326] [Actions]
### Select Topic Area

Bug

### Body

We've been having problems with Google Cloud Build Triggers in connection with our repositories since yesterday afternoon (CET). These are triggered by a push to the branch. We connected about 200 build triggers and got the message: **Some checks haven't completed yet.** **Queued - Waiting to run this check** - and has been for hours now.

**Comment by ossanna16** [comment]
Hi there @inoovum and welcome to our community! Thank you for reporting this bug!
